### ISSUE TITLE
Marina : workflows : 2 weeks

### ISSUE THREAD

**Issue by marina-bashlak** [HYF-Class21/home#40]
> [my fork of module repository]()

## Study checklist

- [x] using basics CLI
- [x] learning basics Git and GitHub
- [x] practicing working  in VSC
- [x] installing NPM and node.js
- [x] learning NPM and node.js

### Week 1

- [ ] I have pushed my progress to [my fork of exercises repo](_)

## I Need Help With:
- "my fork of exercises repo" , from the template message above. What is it about? Did we need to create a fork of some repo?
- it was mentioned that bio has to have the same name as my git name, but my git name has a dash in it, hence I named it just "marina.md" , is it ok?
- tasks says that "My bio link in the class README opens to my student bio file" , could you please explain what it means?

## What went well?
It was a bit overwhelming as for a beginner, but I feel like I'm catching up.

## What went less well?
Sometimes it's hard to understand what to focus about and what details are not that important at this moment.

## Lessons Learned
Basic ideas and usage of GIT and GitHub, working with VCS.
## Sunday Prep Work
Continue reading about NPM, Node.js and git. 
Do I need to prepare anything specific ?

___

> 

### Week 2

- [ ] I have pushed my progress to [my fork of exercises repo](_)
- [x] learning and using collaborative basics and settings
- [x] adding everyone in my team as a contributor with Admin access
- [x] protecting Master from changes
 - [x] creating my group's deliverable issue and placing it on the class project board
 - [x] using a Project Board to organize a project's issues
  - [x] setting  up our group repo's project board
  - [x] link a PR to my claimed issue
  - [x] assigning someone to review my PR
   - [x] fixing merge conflicts by selecting which change to keep, or deciding how to combine them.
   - [ ]  if a conflict occurs in a GitHub PR, pulling both branches and fixing the conflict locally.

**Comment by samirm00** [comment]
@marina-bashlak ,
1- "my fork of exercises repo" , from the template message above. What is it about? Did we need to create a fork of some repo?
2- it was mentioned that bio has to have the same name as my git name, but my git name has a dash in it, hence I named it just "marina.md" , is it ok?
3- tasks says that "My bio link in the class README opens to my student bio file" , could you please explain what it means?


1- `fork a repo`: When you fork a repository, you make a copy of the original repository, but it is stored in your own account , for example, if I like a repo and I want to add something to it I can't because I am not authorized ( not collaborator ), so I can fork it and I will have a copy of it on my account and I can add whatever I want.

2- no it is not ok, because it must be exactly as your GitHub user name even if it has dash init.

3- You don't have to worry about that, because we will add your info to the `README.md` file.

**Comment by marina-bashlak** [comment]
Thank you, @samirm00 , renamed and created PR. Regarding 1, as I understood, it was just an exercises then? I've done it for template-markdown as you mentioned during the lesson that we can't create anything in that repository.

**Comment by samirm00** [comment]
@marina-bashlak , yes, because you didn't have access to `template-markdown` , that is why I asked you to create your own fork.
